### PR TITLE
feat: add tracing

### PR DESCRIPTION
**Pull request by rkok98**

### CODE REVIEW

File: src/tasks/handlers/create-task-handler.ts
Comment (Ryukishi) [inline src/tasks/handlers/create-task-handler.ts:1]: Forbidden non-null assertion - we should look into this

File: test/util/handler-utils.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const mockContext: Context = {
   logGroupName: '',
   logStreamName: '',
   getRemainingTimeInMillis: function (): number {
-    throw new Error('Function not implemented.');
+    return 0;
```
Comment (Ryukishi) [inline test/util/handler-utils.ts:50]: why does this return 0?

**Review by Ryukishi — APPROVED** [review]

File: test/util/handler-utils.ts (hunk 1 above)
Comment (rkok98) [reply test/util/handler-utils.ts:50]: To disable Middy timeout during testing. See: https://middy.js.org/docs/intro/testing/

File: src/tasks/handlers/create-task-handler.ts
Comment (rkok98) [reply src/tasks/handlers/create-task-handler.ts:1]: Will do, out of scope for this PR.

**Pull request merged**